packages/@webex/plugin-meetings/src/meeting/voicea-meeting.ts

```diff
@@ -35,39 +35,6 @@ export const processNewCaptions = ({data, meeting}) => {
   const {transcriptId} = data;
```

Comment:
Do we have any type definition of this `data`?

Reply:
We have it in the tests as sample data.

Reply:
@mkesavan13 why aren't we defining a type for this accordingly to be used in src as well? It would be easier to understand the code.